.golangci.yml

```diff
@@ -29,10 +30,11 @@ issues:
 linters-settings:
   errcheck:
     exclude-functions:
+      # Don't flag lines such as "io.Copy(io.Discard, resp.Body)".
+      - io.Copy
+      - io.WriteString
       # Any error in HTTP handlers is handled by the server itself.
       - (net/http.ResponseWriter).Write
-      # Never check for logger errors.
```

Comment:
I didnt understand why we are removing this.

Reply:
Because `sloglint` takescare of it?

Reply:
> Because `sloglint` takescare of it?

Yes. Replace it with `sloglint`